### DISCUSSION TITLE
Feature Request: Manually selecting which Loki labels to display

### DISCUSSION THREAD

**Discussion by darintay** [grafana/grafana#42764] [Observability]
It would be great to be able to manually pick which labels to display for Loki log data.

Right now it's either all (unique) or none, and all can often be quite noisy if you have a lot of static labels.
 
This looks to have been brought up previous in https://github.com/grafana/grafana/issues/20732 but only partially implemented (without the manual part, unless I'm missing something).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
